src/main/java/com/openshift/internal/restclient/ApiTypeMapper.java

```diff
@@ -141,12 +142,12 @@ private void addEndpoints(final String prefix, final String apiGroupName, final
 			}
```

Comment:
That is correct. Based on the APIs I've seen the names which have these
slashes are only additional actions beyond the typical REST calls. I
presume we would represent them as capabilities

On Tuesday, June 21, 2016, André Dietisheim <notifications@github.com>
wrote:

> In src/main/java/com/openshift/internal/restclient/ApiTypeMapper.java
> <https://github.com/openshift/openshift-restclient-java/pull/176#discussion_r67821509>
> :
>
> > @@ -141,12 +142,12 @@ private void addEndpoints(final String prefix, final String apiGroupName, final
> >  			}
>
> what if the name doesnt contain FWD_SLASH? we wont create a capability? is
> this right?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/openshift/openshift-restclient-java/pull/176/files/f59c9a1f9701ad5c80b57bb356e385de4b64c6a1#r67821509>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AEVnOOya16NnmwlZpw0bOhHsPFrVhsjuks5qN5ZQgaJpZM4I6ToV>
> .
>


-- 
--
Jeff Cantrill
Senior Software Engineer, Red Hat Engineering
OpenShift Integration Services
Red Hat, Inc.
*Office*: 703-748-4420 | 866-546-8970 ext. 8162420
jcantril@redhat.com
http://www.redhat.com